packages/checkout-ui-extensions/README.md

```diff
@@ -0,0 +1,38 @@
+# `@shopify/checkout-ui-extensions`
+
+This library contains type definitions and other utilities for writing UI extensions in Shopify’s checkout. However, nothing in this library is strictly required to write a UI extension. As a developer building a UI extension, you can forego the utilities this library provides entirely, and instead use the [global `shopify` API](documentation/globals.md) directly in a plain JavaScript file. The script below is a valid UI script without any additional processing:
```

Comment:
```suggestion
This library contains type definitions and other utilities for writing UI Extensions in Shopify’s checkout. However, nothing in this library is strictly required to write a UI extension. As a developer building a UI extension, you can forego the utilities this library provides entirely, and instead use the [global `shopify` API](documentation/globals.md) directly in a plain JavaScript file. The script below is a valid UI script without any additional processing:
```

(Maybe I'm wrong about the capitalization here? Not sure 😄)

Reply:
I think this is a named feature so it should be capitalized? We can always update later. I think it's better to be consistent